Bio/SeqUtils/MeltingTemp.py

```diff
@@ -528,40 +567,50 @@ def salt_correction(Na=0, K=0, Tris=0, Mg=0, dNTPs=0, method=1, seq=None):
     if method == 5:
         corr = 0.368 * (len(seq) - 1) * math.log(mon)
     if method == 6:
-        corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(mon) +\
-            9.40e-6 * math.log(mon) ** 2
+        corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(
+            mon
+        ) + 9.40e-6 * math.log(mon) ** 2
```

Comment:
Noooo! There must be a different type of reformatting this formula. But this is really ugly and nonsense.

Reply:
That is rather a mess, I wonder if additional brackets might make us and black happy?

Reply:
black switched off for this.

Reply:
Did adding brackets not help?

Reply:
- For the current  style
```
    if method == 6:
        corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(mon) +\
            9.40e-6 * math.log(mon) ** 2
```
- Black produces
```
    if method == 6:
        corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(
            mon
        ) + 9.40e-6 * math.log(mon) ** 2
```
- Adding brackets around math.log
```
    if method == 6:
        corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * (math.log(mon)) +\
            9.40e-6 * math.log(mon) ** 2
```
- Black produces
```
    if method == 6:
        corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * (
            math.log(mon)
        ) + 9.40e-6 * math.log(mon) ** 2
```
- Adding brackets around the first addend
```
    if method == 6:
        corr = ((4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(mon)) +\
            9.40e-6 * math.log(mon) ** 2
```
- Black produces
```
    if method == 6:
        corr = (
            (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(mon)
        ) + 9.40e-6 * math.log(mon) ** 2
```

Making this 2 lined formula 3 lines and removing the addition operator from the end of the first line, makes it a bit odd to read but if it needs to change let me know your choice.

Reply:
I prefer the final version, Markus?

Reply:
Agree

Reply:
updated.